### PR TITLE
next() function should be used in python3

### PR DESCRIPTION
**Pull request by vanya-antonov**
Added separate usage info for python3:
https://stackoverflow.com/a/1073407/310453

### CODE REVIEW

**Comment by mdehoon** [comment]
I believe the `next` function is also available in Python 2.7.

**Comment by peterjc** [comment]
Well spotted, but as noted we can just use the ``next(...)`` function on both Python 2.7 and 3.

We fixed this on the Tutorial and docstrings a while ago, but having separate documentation on the website has downsides like this.

File: wiki/SeqIO.md [hunk 1]
```diff
@@ -204,11 +204,11 @@ record = SeqIO.read("single.fasta", "fasta")
 
 For the related situation where you just want the first record (and are
 happy to ignore any subsequent records), you can use the iterator's
-`.next()` method:
+`next()` function:
```
Comment (peterjc) [inline wiki/SeqIO.md]: We can probably remove the word "iterator's" as well here?

**Comment by peterjc** [comment]
Looks good, thank you 👍

**Pull request merged**